kuka_lbr_iiwa_moveit_config/srdf/lbr_iiwa_arm.macro.xacro

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <xacro:macro name="kuka_lbr_iiwa_arm" params="prefix">
+    <!--GROUPS: Representation of a set of joints and links. This can be useful for specifying DOF to plan for, defining arms, end effectors, etc-->
+    <!--LINKS: When a link is specified, the parent joint of that link (if it exists) is automatically included-->
+    <!--JOINTS: When a joint is specified, the child link of that joint (which will always exist) is automatically included-->
+    <!--CHAINS: When a chain is specified, all the links along the chain (including endpoints) are included in the group. Additionally, all the joints that are parents to included links are also included. This means that joints along the chain and the parent joint of the base link are included in the group-->
+    <!--SUBGROUPS: Groups can also be formed by referencing to already defined group names-->
+    <group name="lbr_iiwa_arm">
+      <chain base_link="${prefix}base_link" tip_link="${prefix}flange" />
+    </group>
```

Comment:
I just happen to notice it in this `.srdf`, and it's really just something to consider, but at least in ROS-Industrial, we've tried to always make the default `chain` run from `base_link` to `tool0` and call it `manipulator`.

FK/IK would then probably match with what the TP shows, and by using the generic `manipulator` name, switching between manipulators (even from the same OEM) would not require updating references 'everywhere'.

Reply:
good practice, I will fix this too :)